addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,42 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
```

Comment:
We usually add a blank line between the `_fields` and the actual fields definitions.

Reply:
Done